boards/esp32s2-wemos-mini/esp32s2-wemos-mini.config

```diff
@@ -0,0 +1,2 @@
+CONFIG_PACKAGE_TINYUSB=y
```

Comment:
This is again exactly the problem I tried to solve in PR #19038 for the boards with only one USB interface. Using such an application configurations in the board definition prevents switching between the stacks. PR #19086 provides an approach that could solve the problem, by selecting the stack implementation that should be used first and the modules of the according stack afterwards.

But let's continue as is for now.